src/project/FileViewController.js

```diff
@@ -120,14 +107,9 @@ define(function (require, exports, module) {
     /**
      * Modifies the selection focus in the project side bar. A file can either be selected
      * in the working set (the open files) or in the file tree, but not both.
-     * @param {String} fileSelectionFocus - either PROJECT_MANAGER or WORKING_SET_VIEW
+     * @param {String} fileSelectionFocus - PROJECT_MANAGER
      */
     function setFileViewFocus(fileSelectionFocus) {
-        if (fileSelectionFocus !== PROJECT_MANAGER && fileSelectionFocus !== WORKING_SET_VIEW) {
-            console.error("Bad parameter passed to FileViewController.setFileViewFocus");
-            return;
-        }
```

Comment:
Keep this `if` check, since we don't want to break assumptions elsewhere in the code.  However, after it do this:

```js
// XXXBramble: always keep focus on the Project Manager vs. Working Set View
fileSelectionFocus = PROJECT_MANAGER;
```